R/unit.conversions.R

```diff
@@ -273,6 +278,10 @@ flowconcToFluxConversion <- function(flow.units, conc.units, load.rate.units, at
   ## Code inspired by rloadest::loadConvFactor code by DLLorenz and ldecicco
   ## Makes heavy use of unitted package by A Appling
   
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  numerator=denominator=NULL
```

Comment:
this is fixed here https://github.com/miguelcleon/loadflex/blob/master/R/unit.conversions.R#L290